README.rst

```diff
@@ -339,6 +339,17 @@ Choose from the list of available rules:
 
   *Risky rule: forcing strict types will stop non strict code from working.*
 
+* **declare_strict_types_position**
+
+  Move the strict type declaration to the configured location. Requires
+  PHP >= 7.0.
+
+  Configuration options:
+
+  - ``relocate_to`` (``'next'``, ``'same'``): whether ```declare(strict_types=1)````` should
```

Comment:
Not my bug?
<img width="1196" alt="screen shot 2017-07-29 at 12 52 22" src="https://user-images.githubusercontent.com/3288888/28744630-cecd4ea6-745c-11e7-9081-65d1b80348aa.png">